blocks.py

```diff
@@ -1,12 +1,14 @@
 from .multiple import MultipleSignals
 from .generators.counter import CounterGenerator
+from .generators.range import RangeGenerator
 from .generators.identity import IdentityGenerator
 from .triggers.interval import IntervalTrigger
+from .triggers.limit import LimitTrigger
 from .triggers.safe import SafeTrigger
 from .triggers.fast import FastTrigger
 from nio.common.block.base import Block
 from nio.common.discovery import Discoverable, DiscoverableType
-from nio.metadata.properties import VersionProperty
+from nio.metadata.properties.version import VersionProperty
```

Comment:
Good call, thanks. This will be enabled in `1.5.2` but we should use `.version` for now